UnoCheck/RuntimePlatform/OperationalSystem/WindowsRuntime.cs

```diff
@@ -0,0 +1,48 @@
+﻿using DotNetCheck.RuntimePlatform.Core;
+using DotNetCheck.RuntimePlatform.Model;
+using Microsoft.Win32;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DotNetCheck.RuntimePlatform.OperationalSystem
+{
+	public class WindowsRuntime : RuntimePlatformBase
+	{
+		public override IList<InstalledProgramInfo> GetInstalledPrograms()
+		{
+			if (_installedProgramsStore == null)
+			{
+				_installedProgramsStore = new List<InstalledProgramInfo>();
+			}
+			else
+			{
+				return _installedProgramsStore;
+			}
+
+			var machine = Environment.MachineName;
+
+			var registry_key = @"SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall";
+
+			using (RegistryKey key = RegistryKey.OpenRemoteBaseKey(RegistryHive.LocalMachine, machine).OpenSubKey(registry_key))
+			{
+				foreach (string subkey_name in key.GetSubKeyNames())
+				{
+					using (RegistryKey subkey = key.OpenSubKey(subkey_name))
+					{
+						if (subkey.GetValue("DisplayName") != null)
```

Comment:
I wonder, DisplayName may be localized. Isn't there an "ID" or Guid we can use instead?

Reply:
GTK is an acronym, no worries with localization.